kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx

```diff
@@ -12,17 +12,18 @@ import { ColumnDef } from '@tanstack/react-table';
 import { clusterBrokerPath } from 'lib/paths';
 import Tooltip from 'components/common/Tooltip/Tooltip';
 import ColoredCell from 'components/common/NewTable/ColoredCell';
+import { BrokerDiskUsage } from 'generated-sources';
 
 import SkewHeader from './SkewHeader/SkewHeader';
 import * as S from './BrokersList.styled';
 
-const NA = 'N/A';
+const NA = 'N/A' as unknown as number;
```

Comment:
I don't like that we cast string to number

Reply:
It's because in this component we use N/A as a failsafe for a number types. The other way would be to change autogenerated types to also accept string for size and count props

Reply:
we can actually write sanitizer function and link it to `useBrokers` that way the FE will have control over the type, do it once after we fetch inside the hook `useBrokers` instead of doing it with `useMemo` on every render.